site/docs/reference/Connectors/capture-connectors/mongodb.md

```diff
@@ -44,19 +90,25 @@ MongoDB source connector.
+| `/batchAndChangeStream` | Capture Batch Collections in Addition to Change Stream Collections | Discover collections that can only be batch captured if the deployment supports change streams. Check this box to capture views and time series collections as well as change streams. All collections will be captured in batch mode if the server does not support change streams regardless of this setting. | boolean |                  |
+| `/pollSchedule`         | Default Batch Collection Polling Schedule                          | When and how often to poll batch collections. Accepts a Go duration string like '5m' or '6h' for frequency-based polling or a string like 'daily at 12:34Z' to poll at a specific time (specified in UTC) every day. Defaults to '24h' if unset                                                                 | string  |                  |
 
 #### Bindings
 
-| Property          | Title    | Description     | Type   | Required/Default |
-| ----------------- | -------- | --------------- | ------ | ---------------- |
-| **`/database`**   | Database | Database name   | string | Required         |
-| **`/collection`** | Stream   | Collection name | string | Required         |
+| Property          | Title            | Description                                                                                                                                                                                                                                                                                | Type   | Required/Default |
+|-------------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|------------------|
+| **`/database`**   | Database         | Database name                                                                                                                                                                                                                                                                              | string | Required         |
+| **`/collection`** | Stream           | Collection name                                                                                                                                                                                                                                                                            | string | Required         |
+| `/captureMode`    | Capture Mode     | Either **Change Stream Incremental**, **Batch Snapshot**, or **Batch Incremental**                                                                                                                                                                                                         | string |                  |
```

Comment:
I think **Change Stream Incremental** is the default if not set, right? Could that be represented in the "Required/Default" column?

Reply:
Like above, there is no default, although an empty input is handled as "Change Stream Incremental" by the connector.

Reply:
Would it be too verbose to add "Defaults to Change Stream Incremental if unset"?

Reply:
Practically speaking it is impossible to have it be unset going forward, since discovery will always populate it with something, and you won't be able to select an empty input in the UI. The change-stream-if-empty behavior is only there for legacy tasks which do not have anything set.

Reply:
Gotcha, that makes sense! Thanks for explaining that for me!